Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -107,30 +105,20 @@ public class NavigationCameraStateTransition: CameraStateTransition {
     }
     
     public func updateForFollowing(_ cameraOptions: CameraOptions) {
-        // Check whether the original and target locations and headings are larger than a certain threshold when current camera state is following.
-        guard let mapView = mapView,
-              let center = cameraOptions.center,
-              let zoom = cameraOptions.zoom,
-              let bearing = cameraOptions.bearing else { return }
-
-        let centerUpdateThreshould = baseCenterUpdateThreshould * pow(2, 22.0 - Double(zoom))
-        guard mapView.cameraState.center.distance(to: center) > centerUpdateThreshould,
-              abs(mapView.cameraState.bearing - bearing) > bearingUpdateThreshold else { return }
-        
-        update(cameraOptions)
+        update(cameraOptions, state: .following)
     }
     
     public func updateForOverview(_ cameraOptions: CameraOptions) {
         var cameraOptions = cameraOptions
         cameraOptions.bearing = 0.0
-        update(cameraOptions)
+        update(cameraOptions, state: .overview)
```

Comment:
I wonder if we need these helper methods anymore. We could make `update(_:state:)` public and move the `cameraOptions.bearing = 0.0` statement into a conditional in that method.

Reply:
I replaced the two helper methods with the `update(to:state:)`.